frontend/src/metabase/visualizations/components/EmptyVizState/EmptyVizState.tsx

```diff
@@ -59,13 +59,16 @@ export const EmptyVizState = ({
       justify="center"
       data-testid="visualization-placeholder"
     >
-      <Box maw="20rem" mb="3rem">
-        <img
-          src={imgSrc}
-          alt={c("{0} refers to the chart type")
-            .t`${emptyVizChart} chart example illustration`}
-        />
-      </Box>
+      {imgSrc && (
```

Comment:
I guess this is just a precaution (or maybe I'm missing a bit here), but why do we even need this condition if we know for sure that the empty state will not even render in the SDK scenario?

Reply:
I decided to still put the EmptyVizState component itself to the bundle just to cover the case if the condition above may be changed and the EmptyVizState may be displayed in the SDK. But we still won't display the SVG image in that case.

Reply:
I would check with the product and the design teams about this decision first.